java-components/domainproxy/.gitignore

```diff
@@ -0,0 +1,45 @@
+#Maven
+target/
+pom.xml.tag
+pom.xml.releaseBackup
+pom.xml.versionsBackup
+release.properties
+.flattened-pom.xml
+
+# Eclipse
+.project
+.classpath
+.settings/
+bin/
+
+# IntelliJ
+.idea
+*.ipr
+*.iml
+*.iws
+
+# NetBeans
+nb-configuration.xml
+
+# Visual Studio Code
+.vscode
+.factorypath
+
+# OSX
+.DS_Store
+
+# Vim
+*.swp
+*.swo
+
+# patch
+*.orig
+*.rej
+
+# Local environment
+.env
+
+# Plugin directory
+/.quarkus/cli/plugins/
+
+output/
```

Comment:
I think this (and for that matter .mvn directory as well) shouldn't be migrated?

Reply:
We do at least need the 'output' directory to be ignored.

Reply:
We should not be megrating the .gitignore, if there is something missing add it to the top level one.

Reply:
Frankly our top level .gitignore is a bit of a mess with 'dumping ground' of possibly wanted ignores and possibly historic ones. I'd vote for cleaning it up in a separate PR IMHO.